camayoc/tests/remote/__init__.py

```diff
@@ -1,2 +0,0 @@
-# coding=utf-8
```

Comment:
does `remote` module contain only `rho`? If not I'd keep this module

Reply:
It only contains `rho`.